ui/nuxeo-data-table/data-table-row.js

```diff
@@ -26,6 +26,12 @@ import '@nuxeo/nuxeo-elements/nuxeo-element.js';
             user-select: none;
           }
 
+          :host([header]) {
+            border: none;
+            padding-bottom: 0;
+            border-bottom: none;
```

Comment:
I assume this is not needed because of the `border: none;` above?

Reply:
Indeed, you are right. I'm changing it to initial to make it clear that we want the default value.